code/modules/clothing/neck/ties.dm

```diff
@@ -0,0 +1,185 @@
+/obj/item/clothing/neck/tie
+	name = "tie"
+	desc = "A neosilk clip-on tie."
+	icon = 'icons/obj/clothing/neck.dmi'
+	icon_state = "bluetie"
+	item_state = ""	//no inhands
+	item_color = "bluetie"
+	w_class = 2
+
+/obj/item/clothing/neck/tie/blue
+	name = "blue tie"
+	icon_state = "bluetie"
+	item_color = "bluetie"
+
+/obj/item/clothing/neck/tie/red
+	name = "red tie"
+	icon_state = "redtie"
+	item_color = "redtie"
+
+/obj/item/clothing/neck/tie/black
+	name = "black tie"
+	icon_state = "blacktie"
+	item_color = "blacktie"
+
+/obj/item/clothing/neck/tie/horrible
+	name = "horrible tie"
+	desc = "A neosilk clip-on tie. This one is disgusting."
+	icon_state = "horribletie"
+	item_color = "horribletie"
+
+/obj/item/clothing/neck/stethoscope
+	name = "stethoscope"
+	desc = "An outdated medical apparatus for listening to the sounds of the human body. It also makes you look like you know what you're doing."
+	icon_state = "stethoscope"
+	item_color = "stethoscope"
+
+/obj/item/clothing/neck/stethoscope/attack(mob/living/carbon/human/M, mob/living/user)
+	if(ishuman(M) && isliving(user))
+		if(user.a_intent == "help")
+			var/body_part = parse_zone(user.zone_selected)
+			if(body_part)
+				var/their = "their"
+				switch(M.gender)
```

Comment:
I didn't do this, I was copy/pasting shit from the other slots just to be certain on things I was unsure about.